l1-contracts/contracts/state-transition/chain-deps/ZkSyncHyperchainStorage.sol

```diff
@@ -151,4 +151,10 @@ struct ZkSyncHyperchainStorage {
     uint128 baseTokenGasPriceMultiplierDenominator;
     /// @dev The optional address of the contract that has to be used for transaction filtering/whitelisting
     address transactionFilterer;
+    /// @dev The address of the l1DAValidator contract. 
+    /// This contract is responsible for the verification of the correctness of the DA on L1.
+    address l1DAValidator;
```

Comment:
I just want to mention that there are two names Data Availability and Data Publishing. DP id more accurate and some people started to use it, but DA is more common. 

Just so that we know.